nixos/tests/shattered-pixel-dungeon.nix

```
@@ -0,0 +1,29 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "shattered-pixel-dungeon";
+  meta = with pkgs.stdenv.lib.maintainers; {
```

lib is added to stdenv for convenience, but i would avoid it if possible
```suggestion
  meta = with pkgs.lib.maintainers; {
```

---

oops, right. fixed.